app/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AppConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
Why is this configuration made?

Reply:
I don't remember making this if I'm honest, but I've removed it now and appears to be working fine.